app/controllers/emails_controller.rb

```diff
@@ -23,6 +24,11 @@ def submit
     end
   end
 
+  def submit_generic
+    @send_generic = true
```

Comment:
Those are all valid considerations, but what I meant is: why do you set it here in this method? :smile: 
`submit` is called by the view and doesn't render anything....
Oh wait, I understood while I was writing this. `submit` calls `send_email`, which could render `email`. Maybe you could make it clearer that the variable is for that purpose by moving the assignment to `send_email`, just before the call to `render`?